src/components/molecules/BarChartView.tsx

```diff
@@ -1,26 +1,28 @@
 import React, { FunctionComponent } from 'react'
 import { ResponsiveContainer, BarChart, LabelList, XAxis, Label, Bar } from 'recharts'
-import { IWidgetAccidentsByYear } from '../../models/WidgetData'
+import { IWidgetAccidentsByYear, IWidgetInjuredByYear } from '../../models/WidgetData';
+
 interface IProps {
-	data: IWidgetAccidentsByYear[]
+	data: IWidgetAccidentsByYear[] | IWidgetInjuredByYear[]
```

Comment:
since all you do is passing it to rechard, simply give it rechart pie-data type - `ReadonlyArray<BarData>`